### ISSUE TITLE
Some additional settings should be revealed to the user during first-time start

### ISSUE THREAD

**Issue by Hectate** [Jazcash/bar-lobby#712]
We recently added a language selection dropdown as a "step zero" for InitialSetup. For accessibility reasons, we should evaluate if there are other settings that should be exposed to the user earlier in the startup process for a first-time start.

The specific example provided in Discord was "Allow me to mute music volume before the music starts playing". Since the Settings Modal is not available yet, users cannot set their preferences.
